pkg/steps/pod.go

```diff
@@ -285,7 +290,10 @@ func (s *podStep) generatePodForStep(image string, containerResources coreapi.Re
 	}
 
 	artifactDir := s.name
-	pod, err := GenerateBasePod(s.jobSpec, s.config.Labels, s.config.As, s.config.NodeName, s.name, []string{"/bin/bash", "-c", "#!/bin/bash\nset -eu\n" + s.config.Commands}, image, containerResources, artifactDir, s.jobSpec.DecorationConfig, s.jobSpec.RawSpec(), secretVolumeMounts, clone)
+	pod, err := GenerateBasePod(s.jobSpec, s.config.Labels, s.config.As,
+		s.config.NodeName, s.name, []string{"/bin/bash", "-c", "#!/bin/bash\nset -eu\n" + s.config.Commands},
+		image, containerResources, artifactDir, s.jobSpec.DecorationConfig, s.jobSpec.RawSpec(),
+		secretVolumeMounts, &GeneratePodOptions{Clone: clone, PropagateExitCode: false})
```

Comment:
A minor note: here and in other places, `false` is the zero value of `bool`, so statements such as `PropagateExitCode: false` could have been omitted.

Reply:
Okay, I am coming from C world and I am more used to explicitly expressing, not used to that. I will try to remember!